README.md

```diff
@@ -5,27 +5,51 @@
 [![Test Coverage](https://codeclimate.com/github/18F/federalist/badges/coverage.svg)](https://codeclimate.com/github/18F/federalist/coverage)
 [![Dependency Status](https://gemnasium.com/badges/github.com/18F/federalist.svg)](https://gemnasium.com/github.com/18F/federalist)
 
-***Under active development. Everything is subject to change. Learn more at the [documentation site](https://federalist-docs.18f.gov/). Interested in talking to us? [Join our public chat room](https://chat.18f.gov/).***
+***Federalist is updated regularly.  Contact us via [email](federalist-support@gsa.gov) or [Join our public chat room](https://chat.18f.gov/) to talk to us and stay informed. Or, check our our [documentation](https://federalist-docs.18f.gov/).***
 
-Federalist is a unified interface for publishing static government websites. It automates common tasks for integrating GitHub and Amazon Web Services to provide a simple way for developers to launch new static websites or more easily manage existing static websites. This repo is home to "federalist-core" - a Node.js app that allows government users to add and configure their Federalist sites.
+## About Federalist
+
+Federalist helps government agencies publish basic websites quickly and seamlessly.  It provides a simple interface for managing simple website content that is compliant with federal guidelines.  More complex tasks are automated behind the scenes, providing a way for users to more easily create and launch government websites.  Federalist is built on top of Github and Amazon Web Services.   
```

Comment:
I'm not sure the terms `basic` or `simple` are necessary or wanted. Is the issue that `static` is unclear? It is kind of a nuanced technical term, but I'm not sure of a better word for it.

Reply:
`Github` -> `GitHub` (that's how the company stylizes it)

Reply:
I wonder if we should also mention cloud.gov? 

`Federalist is built on top of cloud.gov, Amazon Web Services, and GitHub.`

Reply:
My thought here is that not all users may understand what 'static' means, as not all users will necessarily have a technical background.   Because we'd (I think) want to appeal to the broadest audience possible, more generic terms might be useful.  We could however certainly figure out another place to delineate static.  

👍 to Github styling, and mentioning cloud.gov however!

Reply:
This is a README and not intended as a sales document. It seems like we shouldn't sacrifice any technical detail here IMHO - which user are we writing for?

Reply:
Based on @wslack's most recent comment and my understanding of your users, I think static works here, especially if you define what static means. I'm also making a few copy edits to enhance the language (for example, not using the word simple twice in one sentence). Also adding cloud.gov and editing GitHub style.